app/frontend/src/cljs/teet/project/land_controller.cljs

```diff
@@ -44,6 +45,11 @@
                                   (not (:selected? unit)))))
     cad-units))
 
+(defn cadastral-unit-dom-id [id]
```

Comment:
these 2 should probably be private (`defn-`)

Reply:
There might be a problem with warning like ".. var: teet.project.land-controller/estate-dom-id is not public .." then those functions called from land_view.cljs

Reply:
My bad, I thought these were new and only used here.